pcs/resource.py

```diff
@@ -1501,6 +1563,7 @@ def resource_group_add_cmd(lib, argv, modifiers):
 
     group_name = argv.pop(0)
     resource_names = argv
+    _check_is_not_stonith(lib, resource_names)
```

Comment:
Can this be moved to a library command? Then we could simply replace deprecation warning with an error in order to actually forbid moving stonith resources.